project/src/main/java/com/google/sps/data/GeoContext.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.maps.GeoApiContext;
+
+/**
+ * GeoContext holds the entry point for a Google GEO API request.
+ * It is initiated once and used for all Google GEO EPI requests,
+ * including Google Places API and Google Distance Matrix API.
+ */
+public final class GeoContext {
```

Comment:
I think there's one layer too many here. 

Why do you need an instance of `GeoContext` to hold the `GeoApiContext`? You can have the `GeoApiContext` as the `singleInstance` directly, and thus simplify this class.

WDYT?

Reply:
Agreed, I changed the class a bit to get rid of that layer. Let me know if that's what you meant.